lint/linter_test.go

```diff
@@ -109,3 +110,29 @@ func TestLinter_run(t *testing.T) {
 		}
 	}
 }
+
+type panickingRule struct{}
+
+func (r *panickingRule) Info() RuleInfo                    { return RuleInfo{Name: "panic"} }
+func (r *panickingRule) Lint(_ Request) ([]Problem, error) { panic("panic") }
+
+func TestLinter_LintProtos_RulePanics(t *testing.T) {
+	r, err := NewRules(&panickingRule{})
+	if err != nil {
+		t.Fatalf("Failed to create Rules: %q", err)
+	}
+
+	fd := new(descriptorpb.FileDescriptorProto)
+	fd.SourceCodeInfo = new(descriptorpb.SourceCodeInfo)
```

Comment:
Do we need this line?

Reply:
Yes, otherwise creating request will fail.